lib/metalsmith.js

```diff
@@ -77,7 +81,13 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
   .use(markdown({
     smartypants: true,
     gfm: true,
-    tables: true
+    tables: true,
+    pedantic: {
+      ignoreHtml: true,
+      outdent: true,
+      code: true,
+      inline: true
```

Comment:
Wonder if these arguments should have comments too.

Reply:
Added a global comment for the _pedantinc_ attributes